include/cvc5/cvc5_kind.h

```diff
@@ -175,8 +175,8 @@ enum ENUM(Kind)
    * .. note::
    *
    *     Represents an internally generated term. Information on the skolem is
-   *     available via the calls `Solver::getSkolemId` and
-   *     `Solver::getSkolemIndices`.
+   *     available via the calls `Term::getSkolemId` and
+   *     `Term::getSkolemIndices`.
```

Comment:
These don't resolve (https://cvc5.github.io/docs-ci/docs-pr12014/api/cpp/enums/kind.html#_CPPv4N4cvc54Kind6SKOLEME), I think it's because you are missing parenthesis. Maybe this will fix it: 
```suggestion
   *     available via the calls `Term::getSkolemId()` and
   *     `Term::getSkolemIndices()`.
```
If not, then you will also have to add the parameters.

Reply:
The references to `Term::getSkolemId` and `Term::getSkolemIndices` required the `:cpp:func:` annotation because they are used within an RST block. The `TermManager` references did not resolve because they point to methods without the `const` qualifier, whereas the old `Solver` methods include the `const` qualifier.

All references are fixed now.